scripts/packager-codes/non-eu/README.md

```diff
@@ -32,3 +32,9 @@ To display the status of the locally downloaded files as compared to the remote:
 ````shell script
 python packager_codes.py status packager_codes_data
 ````
+
+To extract codes
+````
+find . -name "*.pdf"
+python ./pdf_extraction.py    
```

Comment:
This doesn't do anything with the file list, does it? Does the output need to be piped into python or used as the argument list?

Also, should this note that all of this syntax requires *NIX or at least GNU findutils, or any other specific platform requirements?

Reply:
I'm working on it tonight. Marking as WIP

Reply:
oh ok. this confused me a bit. 😀 

Are you editing this in the GitHub website or do you push it manually?

Reply:
For this PR, from the GitHub web interface. On multifile PRs: commandline